test/suite/test_config04.py

```diff
@@ -203,18 +203,43 @@ def test_eviction_dirty_tgt_abs_too_large(self):
                                  'eviction_dirty_target=1G'),
             '/eviction dirty target should not exceed cache size/')
 
-    def test_eviction_dirty_trigggr_abs_too_large(self):
+    def test_eviction_dirty_trigger_abs_too_large(self):
         self.assertRaisesWithMessage(wiredtiger.WiredTigerError, lambda:
             self.wiredtiger_open('.','create,cache_size=500MB,'
                                  'eviction_dirty_trigger=1G'),
             '/eviction dirty trigger should not exceed cache size/')
 
+    def test_eviction_dirty_trigger_abs_equal_to_dirty_target(self):
```

Comment:
If we disallow equal, this test will go away.

Reply:
We can keep the test but we expect a failure.

Reply:
See f73f0ef48c15d1e123ba047b28f6b56d3a0bb40b